src/Orc.NuGetExplorer/Models/NugetFeed.cs

```diff
@@ -117,6 +122,8 @@ private void RaiseErrorsChanged(string propertyName)
 
         private void ValidateAndRaiseErrorsChanged(string propertyName)
         {
+            _propertyNameToDataError.Clear();
+
             var error = this[propertyName];
 
             if (!_propertyNameToDataError.TryGetValue(propertyName, out string oldError))
```

Comment:
I think it will be always True, because you are learning the dictionary in line 125

Reply:
Hmm, your right, silly, it works perfect on testing. Will update solution

Reply:
Thanks, i was also missing, i uncaressly moved subscription from ctor to InitializeAsync() which is caused some issues too. Fixed.